src/clients/dotnet/TigerBeetle.Tests/IntegrationTests.cs

```diff
@@ -1540,38 +1563,38 @@ private void ConcurrentTasksDispose(bool isAsync)
 
     [TestMethod]
     [ExpectedException(typeof(ObjectDisposedException))]
-    [DoNotParallelize]
     public void DisposedClient()
     {
-        using var server = new TBServer();
-        using var client = GetClient(server.Address);
+        Assert.IsNotNull(server);
+        using var client = new Client(0, new[] { server.Address });
 
+        var accounts = GenerateAccounts();
         var accountResults = client.CreateAccounts(accounts);
         Assert.IsTrue(accountResults.Length == 0);
 
         client.Dispose();
 
         var transfer = new Transfer
         {
-            Id = 1,
+            Id = ID.Create(),
             CreditAccountId = accounts[0].Id,
             DebitAccountId = accounts[1].Id,
             Amount = 100,
             Ledger = 1,
             Code = 1,
         };
 
-        _ = client.CreateTransfers(new Transfer[] { transfer });
+        _ = client.CreateTransfers(new[] { transfer });
         Assert.Fail();
     }
 
 
-    private static void AssertAccounts(Account[] lookupAccounts)
+    private static void AssertAccounts(Account[] expected, Account[] actual)
```

Comment:
Can we use this thing here? 

https://stackoverflow.com/a/11055686

hopefully it’ll print readable diff on mismatxhn

Reply:
We can't compare all fields; for example, the balances can change between creation and lookup.